### PR TITLE
fixed nav links that don't have sub-menus from having right arrow

### PR DESCRIPTION
**Pull request by digitaldrummerj**
added if statement around has-dropdown so If the link does not have a dropdown attribute then the has-dropdown css class is not added.

### CODE REVIEW

File: _includes/navigation_link.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
 	{% if include.link.side == "left" %}<li class="divider"></li>{% endif %}
 {% else %}
 	{% if include.link.side == "right" %}<li class="divider"></li>{% endif %}
-	<li class="has-dropdown{% if page_url == link_url or page_url contains link_url and link_url != '/' %} active{% endif %}">
+	<li class="{% if include.link.dropdown %}has-dropdown {% endif %}{% if page_url == link_url or page_url contains link_url and link_url != '/' %} active{% endif %}">
```
Comment (TWiStErRob) [inline _includes/navigation_link.html:37]: This line is in the `else` of `include.link.dropdown == nil`, so your added condition should always be true.

**Review by TWiStErRob — COMMENTED** [review]
Have you got a an example navigation.yml where this fails?

**Comment by digitaldrummerj** [comment]
Here is a snippet from my navigation that fails.    The Blog should be a drop down but none of the sub items are dropdown.  

```
- title: Blog
  url: "/blog"
  side: left
  dropdown:
  - title: "Blog Archive"
    url: "/blog/archive/"
  - title: "Blog Archive by Date"
    url: "/blog/archive/monthview/"
  - title: "Blog Archive by Category"
    url: "/blog/archive/categoryview"
  - title: "Blog Archive by Tag Cloud"
    url: "/blog/archive/tagcloudview/"

```

**Comment by TWiStErRob** [comment]
I added this block to my `navigation.yml` and the result is as expected:
image

Are you using [up-to-date versions](https://pages.github.com/versions/)?

---

If you want to use my automatically generated date/category/tag views, you have to use a special:
```yml
  - title: "Blog Archive by Tag Cloud"
    url: "/blog/archive/tagcloudview/"
    dropdown:
    - special: blog-tags
```

**Comment by TWiStErRob** [comment]
Thank you for being eager to contribute back, keep up the spirit! I'm going to close this PR as my website looks and works as intended.

**Pull request closed without merging**